### ISSUE TITLE
Checkbox fields not being passed to PipeDrive

### ISSUE THREAD

**Issue by mylesthedev** [verbb/formie#363]
**Description**
Multi-Checkbox fields don't appear to be passed through to Pipedrive via integrations.
All other fields are coming through fine but our checkbox field is not.

The field type in Pipedrive is text so shouldn't be an issue that end.

The values of the checkbox are just single words e.g. curtains, blinds, shutters.

Here's an example of what is being sent via Formie to Pipedrive:
```json
"dealFieldMapping": {
    "641da650ca97b791671f811901ed166d29d881bb": "{tellUsWhatFurnishingYouReAfter}",
}
```
I assume the selected values should be turned into a comma separated string?

**Steps to reproduce**
1. Make form with multi select checkbox
2. Integrate with pipe drive and push checkbox values to field in pipedrive

### Form settings
- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.3.19.1
- Craft version: 3.6.7
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.19.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.20](https://github.com/verbb/formie/releases/tag/1.3.20)
